src/hooks/useWalletsWithBalancesAndNames.ts

```diff
@@ -6,14 +6,14 @@ import { Address } from 'viem';
 
 export default function useWalletsWithBalancesAndNames() {
   const { walletNames, wallets } = useWallets();
-  const walletBalances = useWalletBalances(wallets!);
+  const walletBalances = useWalletBalances(wallets || {});
```

Comment:
can you double check that this will behave as expected for `{}` and that the changes on line 16 won't cause us issues (in `useWalletBalances` will there be issues in passing '[]' empty array for `allAddresses` to `useAddysSummary` etc

Reply:
yes it's fine